nano/node/lmdb.cpp

```diff
@@ -1344,6 +1491,24 @@ std::shared_ptr<nano::block> nano::mdb_store::block_get (nano::transaction const
 		nano::bufferstream stream (reinterpret_cast<uint8_t const *> (value.mv_data), value.mv_size);
 		result = nano::deserialize_block (stream, type);
 		assert (result != nullptr);
+		if (sideband_a)
+		{
+			sideband_a->type = type;
+			if (full_sideband (transaction_a) || entry_has_sideband (value, type))
+			{
+				auto error (sideband_a->deserialize (stream));
+				assert (!error);
+			}
+			else
+			{
+				// Reconstruct sideband data for block.
+				sideband_a->account = block_account_computed (transaction_a, hash_a);
+				sideband_a->balance = block_balance_computed (transaction_a, hash_a);
+				sideband_a->successor = block_successor (transaction_a, hash_a);
+				sideband_a->height = std::numeric_limits<uint64_t>::max ();
+				sideband_a->timestamp = std::numeric_limits<uint64_t>::max ();
```

Comment:
May be set default timestampt to 0?

Reply:
Does anyone else have an opinion on it?  I used height = max as a sentinel because 0 is a valid height. I could set the timestamp sentinel to 0 instead of max, this just matches height.

Reply:
@rkeene @argakiig @cryptocode ?

Reply:
Either is fine with me.  `0` is falsey, which may be useful in some contexts, it's more commonly used to represent "time unavailable", and it's also easier to type.  `::max()` will never be a valid time, even if an unreasonable epoch is used for the system time keeping, and it's consistent with the existing members.